examples_tests/42.FragmentShaderPathTracer/common.glsl

```diff
@@ -14,9 +14,22 @@ layout(set = 3, binding = 0) uniform sampler2D envMap;
 layout(set = 3, binding = 1) uniform usamplerBuffer sampleSequence;
 layout(set = 3, binding = 2) uniform usampler2D scramblebuf;
 
-layout(location = 0) in vec2 TexCoord;
+layout(set=0, binding=0, rgba16f) uniform image2D outImage;
 
-layout(location = 0) out vec4 pixelColor;
+#ifndef _NBL_GLSL_WORKGROUP_SIZE_
+#define _NBL_GLSL_WORKGROUP_SIZE_ 16
+layout(local_size_x=_NBL_GLSL_WORKGROUP_SIZE_, local_size_y=_NBL_GLSL_WORKGROUP_SIZE_, local_size_z=1) in;
+#endif
```

Comment:
AFAIK, this declaration needs to be one of the first things in a compute shader, I'm kinda surprised glslang swallowed it